jekyll/_cci2/migrate-from-launch-agent-to-machine-runner-3-on-linux.adoc

```diff
@@ -3,21 +3,19 @@ contentTags:
   platform:
   - Cloud
 ---
-= Migrate from launch agent to machine runner 3.0 - Open preview
+= Migrate from launch agent to machine runner 3.0 on Linux - Open preview
```

Comment:
I think we should remove "Open preview" here. Similarly for the other migration docs pages.